platform/core/PolyPodCoreAndroid/core/src/main/kotlin/coop/polypoly/core/CoreExceptions.kt

```diff
@@ -9,7 +9,8 @@ enum class CoreExceptionCode(val value: Int) {
     NullCStringPointer(4),
     FailedToCreateCString(5),
     FailedToExtractJavaString(6),
-    FailedToConvertJavaString(7);
+    FailedToConvertJavaString(7),
+    FailedToSetPreference(28);
```

Comment:
This needs to match the enum index in core - seems we didn't add a lot of error codes here, hence the gap.